packages/salesforcedx-vscode-visualforce/package.json

```diff
@@ -23,7 +23,6 @@
   },
   "categories": ["Languages"],
   "devDependencies": {
-    "@salesforce/salesforcedx-utils-vscode": "40.4.0",
```

Comment:
@guw - Was there a reason you forced this one to have a devDependency on salesforcedx-utils-vscode even though it is not being used?